SmartDeviceLinkTests/ProxySpecs/SDLSecondaryTransportManagerSpec.m

```diff
@@ -552,12 +585,32 @@ + (void)swapConnectionMethods {
                     testTransportEventUpdateMessage = [[SDLV2ProtocolMessage alloc] initWithHeader:testTransportEventUpdateHeader andPayload:testTransportEventUpdatePayload.data];
                 });
 
-                it(@"should transition to Connecting state", ^{
-                    [testPrimaryProtocol handleBytesFromTransport:testTransportEventUpdateMessage.data];
-                    [NSThread sleepForTimeInterval:0.1];
+                context(@"before the security manager is set by register app interface response", ^{
+                    it(@"should transition to Connecting", ^{
+                        [testPrimaryProtocol handleBytesFromTransport:testTransportEventUpdateMessage.data];
```

Comment:
This should be sent in the above beforeEach.

Reply:
moved